packages/nextjs/utils/scaffold-eth/parseAddressTo0x.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * @dev Converts an address string to the form of `0x${string}`
+ * @param {string} address - address to be parsed
+ * @return string of the form `0x${string}`
+ */
+export default function parseAddressTo0x(address: string | undefined): `0x${string}` | undefined {
```

Comment:
Added this function because now wagmi only accept address of type `0x{string}` .

- Feel free to suggest a better name 
- also was a bit uncomfortable for creating a whole new file for one function, should we create one general file that contains utils that are **pure**,  or should we directly add them in `index.ts` of `utlis/scaffol-eth` ?

Reply:
but how address can be without 0x?) for me it's ok to create a single file for helper functions, it's easier to find it.

Reply:
also, you can create type something like `0xAddress = 0x{string}` and use it.

Reply:
would love to hear file name suggestion 😅, Thanks 🙌

Reply:
`convertTo0xAddress`?